lazy-imports-mixin.html

```diff
@@ -0,0 +1,71 @@
+<!--
+@license
+Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`lazy-imports`
+<link rel="import" href="./lazy-imports-import.html">
+Declarative lazy imports
+
+@demo demo/index.html
+-->
+<script>
+  (() => {
+    const __loadedImports = {};
+
+    Polymer.LazyImportsMixin = (baseClass) => {
+      return class extends baseClass {
+        importLazyGroup(group, fromElement) {
+          if (!fromElement)
+            fromElement = Polymer.DomModule.import(this.localName);
+
+          let groupAttribute;
+          if (group) {
+            groupAttribute = `[group=${group}]`;
+          } else {
+            groupAttribute = 'not([group])';
+          }
+          const query = `link${groupAttribute}[rel='lazy-import'][href]:not([href=''])`;
```

Comment:
So much of this logic is duplicated with the behavior, can you refactor so that they share some functions?